examples/http3_client.py

```diff
@@ -129,26 +136,39 @@ def __init__(self, *args, **kwargs) -> None:
         else:
             self._http = H3Connection(self._quic)
 
-    async def get(self, url: str, headers: Dict = {}) -> Deque[H3Event]:
+    async def get(self, url: str, headers: Optional[Dict] = None) -> Deque[H3Event]:
         """
         Perform a GET request.
         """
+        if headers is None:
+            headers = {}
+
         return await self._request(
             HttpRequest(method="GET", url=URL(url), headers=headers)
         )
 
-    async def post(self, url: str, data: bytes, headers: Dict = {}) -> Deque[H3Event]:
+    async def post(
+        self, url: str, data: bytes, headers: Optional[Dict] = None
+    ) -> Deque[H3Event]:
         """
         Perform a POST request.
         """
+        if headers is None:
```

Comment:
This seems unnecessary if HttpRequest accepts None

Reply:
Fixed.